hooks/test_post_gen_project.py

```diff
@@ -116,16 +111,36 @@ def test_load_bib_info(
     shutil.copy(manuscript_path, manuscript_in_project)
     Path(project_dir_with_bib / "bib-in-project.bib").touch()
 
-    for file in missing_files:
-        Path(project_dir_with_bib / file).unlink()
-
-    for file in source_dir.iterdir():
-        if file.name in missing_files:
-            assert not (project_dir_with_bib / file.name).exists()
-        else:
-            assert (project_dir_with_bib / file.name).exists()
+    if not exists_bib:
+        for item in project_dir_with_bib.iterdir():
+            if item.isfile() and item.endswith(".bib"):
+                item.unlink()
```

Comment:
instead of all this work, isn't it easier to do something like:
```
project_dir_with_bib = user_filesystem / "project-dir-with-bib"
project_dir_with_bib.mkir()
shutil.copy(manuscript_path, project_dir_with_bib)
manuscript_in_project = project_dir_with_bib / "manuscript.tex"
if exists_bib:
    project_dir_with_bib / "bib1.bib"
    project_dir_with_bib / "bib2.bib"
```
then we have a dir with a valid `manuscript.tex` and bib files which is all we need.